Assets/Mirror/Editor/SceneDrawer.cs

```diff
@@ -10,47 +11,31 @@ public override void OnGUI(Rect position, SerializedProperty property, GUIConten
         {
             if (property.propertyType == SerializedPropertyType.String)
             {
-                SceneAsset sceneObject = GetSceneObject(property.stringValue);
-                SceneAsset scene = (SceneAsset)EditorGUI.ObjectField(position, label, sceneObject, typeof(SceneAsset), true);
-                if (scene == null)
+                SceneAsset sceneObject = AssetDatabase.LoadAssetAtPath<SceneAsset>(property.stringValue);
+
+                if (sceneObject == null)
```

Comment:
```suggestion
                if (sceneObject == null && !string.IsNullOrEmpty(property.stringValue))
```
or add this check at start of `GetBuildSettingsSceneObject`